src/filter/GaussianFilter.js

```diff
@@ -0,0 +1,58 @@
+import IJ from '../ij';
+
+export default function gaussian_filter(n, sigma){
```

Comment:
The "gaussian" filter is a specific application of the convolution filter like probably "blur", "sharpen", ...
The code with therefore "import" the convolution filter and call it. We will show you how to do once the "convolution" filter works.